Readme.md

```diff
@@ -1,76 +1,17 @@
-# Today's Objective: A personal NPM library
+# idkjay personal package
 
-## Prize
+## Description
 
-$25 CAD Amazon gift card + bragging rights.
+Don't leave your messages unread! Wrap them with the eyes emoji using this package. I created this because why not.
+Also includes many react and frontend packages. Also I have no idea what I'm doing.
 
-Tip: Read the [General Information Document](General%20Information.md) for more info on how to get started and submit.
+##  Installation
 
-## Objective
+1. npm i @idkjay/personal
+2. require('@idkjay/personal')
```

Comment:
I think it would be good to add a "how to use" section, like this:

``` const eyes = require('@idkjay/personal');
const unread = 'Please read my message.';
const read = eyes(unread);

console.log(read);```

It would be easier for whoever is installing to use it.